### PR TITLE
Edit newlines across a few files

### PR DESCRIPTION
**Pull request by AlexMapley**
## Background
I copied these charts over to https://github.com/rstudio/platform-labs, to help prepare for a possible Mercury Chart configuration https://github.com/rstudio/platform-labs/pull/3/files.

We'd like to contribute to https://github.com/rstudio/helm now that we're using/copy+pasting the code. Hopefully these newline edits are all welcome, at least as a start.

## Summary
- Edits a few newlines, mostly removing duplicates but also adding in one case

### CODE REVIEW

File: charts/rstudio-launcher-rbac/README.md [hunk 1]
```diff
@@ -55,4 +55,3 @@ helm template -n rstudio rstudio-launcher-rbac rstudio/rstudio-launcher-rbac
 
 ----------------------------------------------
 Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
-
```
Comment (AlexMapley) [inline charts/rstudio-launcher-rbac/README.md:58]: It doesn't really show well on github, but there were 2 newlines at the end of this file and we removed 1 of them.

**Comment by AlexMapley** [comment]
👋 to everyone tagged on my random newline removal PR. Thank you for your time.

**Review by atheriel — APPROVED** [review]
This seems fine and I don't foresee any unintended side effects.

**Comment by colearendt** [comment]
Because we are changing the charts, this will cause version bumps / releases. If you can leave this PR as-is, we have some other PRs that will touch these charts soon that we can pull this into.

**Comment by AlexMapley** [comment]
That's perfect @colearendt, thanks for taking this on I will leave this PR as in then

**Pull request merged**